src/Aspire.ProjectTemplates/README.md

```diff
@@ -0,0 +1,81 @@
+# Aspire Templates
+
+Templates are located at *[repo_root]/src/Aspire.ProjectTemplates/*. The *Aspire.ProjectTemplates.csproj* file builds the template package. The template content is in the *templates* sub-directory, with each sub-directory beneath that representing a separate project template.
+
+## Update instructions
+
+Instructions to update the project templates for a new Aspire version.
+
+Example versions used here are moving from Aspire 9.4 to 9.5. This means:
+
+- Old previous version = `9.3`
+- New previous version = `9.4`
+- Old current version = `9.4`
+- New current version = `9.5`
+
+### Updating template content
+
+Each template has a content folder for the previous version and the current version (`major.minor` only). The content of the previous version is static (i.e. not updated by the build) and will never be updated again after this process. The content of the current version has placeholders for many of the versions represented that get replaced by the build. Note that some dependency versions are still static.
```

Comment:
> wwill never be updated again after this process. 

Presumably unless there's a servicing level bug.

Reply:
Although, perhaps you mean if we service the previous version before we've released the new current version, e.g. we ship 9.4.2 before we ship 9.5.0. In that case, yeah we'd update this.

Reply:
> No, the old version is no longer supported so won't ever be updated. It's included just for convenience sake so that updating to the new templates package doesn't instantly remove ability to work with the previous version.

This seems a bit squishy -- to intentionally set customers up in an unsupported configuratin. Is the assumption they'd not ship their product until they get into a supported configuration? Will they figure that out?